tests/SampleApps/nodejs/webfrontend/server.js

```diff
@@ -10,7 +10,7 @@ app.get('/api', function (req, res) {
     res.send('Hello from webfrontend');
 });
 
-var port = process.env.PORT || 80;
+var port = process.env.PORT || 8080;
```

Comment:
Curious, why this change? Could you paste the contents of `docker inspect` of the final runtime image that the build package produces for this test?

Reply:
this change is because Heroku's generated launch script doesn't set the PORT env var, so **this app would start on port 80, and fail** (_Access Denied_)